addons/mrp/migrations/12.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,120 @@
+---Fields in module 'mrp'---
+mrp          / mrp.bom                  / message_last_post (datetime)  : DEL 
+mrp          / mrp.bom                  / message_main_attachment_id (many2one): NEW relation: ir.attachment
+mrp          / mrp.bom                  / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+mrp          / mrp.bom.line             / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+mrp          / mrp.production           / activity_date_deadline (date) : not related anymore
+mrp          / mrp.production           / activity_date_deadline (date) : not stored anymore
+mrp          / mrp.production           / activity_date_deadline (date) : now a function
+mrp          / mrp.production           / message_last_post (datetime)  : DEL 
+mrp          / mrp.production           / message_main_attachment_id (many2one): NEW relation: ir.attachment
+mrp          / mrp.production           / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+mrp          / mrp.production           / product_uom_qty (float)       : NEW
```

Comment:
Put why nothing to do: computed field. By the way: why this doesn't reflect that is a computed stored field?

Reply:
it's a old analysis_work from an old analysis